### PR TITLE
feat: :sparkles: toKebabCase

### PR DESCRIPTION
**Pull request by aceol**
Hi, this is for #17, really fun (and interesting) to do

### CODE REVIEW

**Comment by aceol** [comment]
I added the aliases + dependency from the plexis package, but it was obviously failing because it's not published. Is it ok or did I miss a step?

File: packages/toKebabCase/src/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * @description Converts a string to kebab case
+ * @param {String} text
+ * @example
+ *  toKebabCase('Cool'); // returns 'cool'
+ *  toKebabCase('cool mate'); // returns  'cool-mate'
+ *  toKebabCase('Hey how are you today?'); // returns  'hey-how-are-you-today'
+ *  toKebabCase('camelCase'); // returns  'camel-case'
+ * toKebabCase('PascalCase'); // returns  'pascal-case'
```
Comment (vorillaz) [inline packages/toKebabCase/src/index.js]: Missing indentation

**Comment by aceol** [comment]
OK, I understand better the process now, thanks for the clear explanation of the review!

**Comment by vorillaz** [comment]
Hello @adriaanvermeire . You must rebase your branch with the master branch in order to merge your work.

**Comment by aceol** [comment]
just did another merge ;)

**Comment by vorillaz** [comment]
@aceol  You should rebase your branch with the upper stream, there is a ton of files in the PR 
More infos can be found [here](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request)

**Pull request closed without merging**